src/api-umbrella/web-app/app/controllers/api/v1/users_controller.rb

```diff
@@ -92,14 +92,12 @@ def update
     end
   end
 
-  def destroy do
+  def destroy
     @api_user = ApiUser.find(params[:id])
+    authorize(@api_user)
```

Comment:
Thanks. I'm not a rails guy and it shows.

Reply:
hehe, the same here, I detected this by looking into the code.